docs/src/references/child-devices.md

```diff
@@ -0,0 +1,62 @@
+   * Events, see [Sending an event to a child-device](../tutorials/send-events.md#sending-an-event-for-a-childexternal-device-to-the-cloud)
+   * Configuration Management, see [Configuration files for child devices](../references/c8y-configuration-management.md#configuration-files-for-child-devices)
+
+## Child-Device Provisioning
+
+The provisioning phase of a child-device includes:
+  1) creating the child-device twin in the cloud
+  2) declaring all supported capabilities on the child-device twin in the cloud
+     
+     (i.E. declaring all _supported operations_ and all _types_ per supported operation, as _config types_, _log types_, ...) 
+
+The user journeys below outline the behaviour of external-devices, thin-edge and the cloud for the provisioning phase.
+
+#### Personas used in user journeys
+* **device-operator**: the one who maintains/operates the device in the field in the shopfloor
+   
+#### User Journey 1: Connecting an external-device as new child-device, provisioning managed by device
+  - device-operator: connects new external-device to the thin-edge device by cable
+  - device-operator: powers up the external-device
+    - the external-device announces it-self to thin-edge with it's _child-id_ and capabilities (e.g. provided _configuration types_, _log types_, ...)
+      Details, see: [_configuration types_](https://github.com/thin-edge/thin-edge.io/blob/cdab5683de9f9e0f34fa42a094ac399f6dbdc1e3/docs/src/references/c8y-configuration-management.md#managing-supported-configuration-list-of-child-devices)
+    - thin-edge creates the child-device twin in the cloud, based on the _child-id_ reported by the external-evice
+
+      NOTE: Cloud's operation to create the child-device twin is idempotent, so trying to create a child-device twin again that already exist does not harm.
+    - thin-edge declares all capabilities reported by the external-device to the child-device twin (i.E. all _supported operations_ and corresponding types per operations, as _config-types_, _log-types_, ...)
+  - from that point the new external-device is ready in operation
+
+#### User Journey 2: Connecting an external-device as new child-device, provisioning managed by a cloud-site backend
+  - device-operator: creates the child-device twin in the cloud using some customer-specific backend; as _child-id_ the device-operator uses some UID that is known by the external-device (e.g. the external device's serial number or network address)
+  - device-operator: connects new external-device to the thin-edge device by cable
+  - device-operator: powers up the external-device
+    - the external-device announces it-self to thin-edge with it's _child-id_ and capabilities (e.g. provided _configuration types_, _log types_, ...)
+    - thin-edge tries to created the child-device twin in the cloud, even though that twin was already created upfront.
```

Comment:
This device creation attempt should be replaced with a device existence check here, right? If the device exists already, thin-edge will have to fetch all the device capabilities from the cloud so that it can keep all that information persisted on its file system.

Reply:
Removed from that PR, but to be considered in PR "requirements and use-cases" (#1195).
See more details in comment https://github.com/thin-edge/thin-edge.io/pull/1236#discussion_r955272147